apps/e2e/src/engine-client/lpTests.ts

```diff
@@ -0,0 +1,57 @@
+import { EngineClient } from '@vertex-protocol/engine-client';
+import { VERTEX_ABIS } from '@vertex-protocol/contracts';
+import { addDecimals } from '@vertex-protocol/utils';
+import { getContract } from 'viem';
+import { prettyPrint } from '../utils/prettyPrint';
+import { RunContext } from '../utils/types';
+import { runWithContext } from '../utils/runWithContext';
+
+async function lpTests(context: RunContext) {
+  const walletClient = context.getWalletClient();
+  const walletClientAddress = walletClient.account.address;
+  const chainId = walletClient.chain.id;
+
+  const client = new EngineClient({
+    url: context.endpoints.engine,
+    walletClient,
+  });
+
+  const clearinghouse = getContract({
+    abi: VERTEX_ABIS.clearinghouse,
+    address: context.contracts.clearinghouse,
+    client: walletClient,
+  });
+
+  const endpointAddr = await clearinghouse.read.getEndpoint();
+
+  const mintSpotLpResult = await client.mintLp({
+    subaccountOwner: walletClientAddress,
+    subaccountName: 'default',
+    productId: 3,
+    amountBase: addDecimals(1),
+    quoteAmountLow: addDecimals(1000),
```

Comment:
Could we change this to use oracle price * 0.5 (for low) & oracle price * 1.5 (for high)?